src/components/molecules/MapCard/defaultStyles.js

```diff
@@ -9,8 +9,9 @@ const defaultStyles = {
     background: "white",
     maxWidth: 345,
     maxHeight: "100%",
-    width: "",
-    height: "100%",
+    height: 526,
```

Comment:
it was necessary to remove width?

Reply:
no is necessary remove, I restored this  width: "",